database/ddl/postgres/build.go

```diff
@@ -11,37 +11,44 @@ const (
 CREATE TABLE
 IF NOT EXISTS
 builds (
-	id            SERIAL PRIMARY KEY,
-	repo_id       INTEGER,
-	number        INTEGER,
-	parent        INTEGER,
-	event         VARCHAR(250),
-	status        VARCHAR(250),
-	error         VARCHAR(500),
-	enqueued      INTEGER,
-	created       INTEGER,
-	started       INTEGER,
-	finished      INTEGER,
-	deploy        VARCHAR(500),
-	clone         VARCHAR(1000),
-	source        VARCHAR(1000),
-	title         VARCHAR(1000),
-	message       VARCHAR(2000),
-	commit        VARCHAR(500),
-	sender        VARCHAR(250),
-	author        VARCHAR(250),
-	email         VARCHAR(500),
-	link          VARCHAR(1000),
-	branch        VARCHAR(500),
-	ref           VARCHAR(500),
-	base_ref      VARCHAR(500),
-	head_ref      VARCHAR(500),
-	host          VARCHAR(250),
-	runtime       VARCHAR(250),
-	distribution  VARCHAR(250),
-	timestamp     INTEGER,
+	id             SERIAL PRIMARY KEY,
+	repo_id        INTEGER,
+	number         INTEGER,
+	parent         INTEGER,
+	event          VARCHAR(250),
+	status         VARCHAR(250),
+	error          VARCHAR(500),
+	enqueued       INTEGER,
+	created        INTEGER,
+	started        INTEGER,
+	finished       INTEGER,
+	deploy         VARCHAR(500),
+	deploy_payload VARCHAR(2000),
```

Comment:
Should we declare a higher limit for this column?

Reply:
Do you have any sample data to compare this value to? 
Or for example, what you might expect to add to go-vela/mock for the build response?
https://github.com/go-vela/mock/blob/master/server/build.go

Reply:
This can technically contain any valid JSON or string if the deployment was created directly via the GitHub API. One example of expected input can be found [here](https://github.com/go-vela/server/pull/217/files#diff-e152bd7186c7835e4de082e909c8df1bb0550503a5bbdcf58e1444034ed69c91R10-R13).